app/scripts/lib/flood/flood.js

```diff
@@ -293,12 +293,12 @@ define('FLOOD', function() {
 								that.markClean();
 
 								if ( that.doPostProcess && that.postProcess ){
-									that.prettyValue = that.postProcess( that.value );
```

Comment:
'prettyValue' is not accurately describes data that stores in it. Changed it to 'geometry'.